### ISSUE TITLE
Stop lockout files from being created

### ISSUE THREAD

**Issue by Robdel12** [thefrontside/.emacs.d#22]
```
Error: ENOENT, no such file or directory '/Users/robertdeluca/Projects/vishnu-frontend/tmp/jshinter-output_path-amVSVkdJ.tmp/vishnu/templates/components/.#objective-assessor.hbs'
```

It's time for these to go away! :P

There was a snippet that @cowboyd sent me `(setq create-lockfiles nil)`, but I'm not sure where this should live?

**Comment by cowboyd** [comment]
What about just adding a `frontside/disable-lockfiles.el`

**Comment by cowboyd** [comment]
All the `.el` files in the `frontside` directory get evaluated.

**Comment by Robdel12** [comment]
I'll create a PR

**Comment by Robdel12** [comment]
This was closed with #23
